irc/config.go

```diff
@@ -491,6 +492,10 @@ func LoadConfig(filename string) (config *Config, err error) {
 	if len(config.Server.Listen) == 0 {
 		return nil, ErrNoListenersDefined
 	}
+	//dan: automagically fix identlen until a few releases in the future (from now, 0.12.0), being a newly-introduced limit
+	if config.Limits.IdentLen < 1 {
+		config.Limits.IdentLen = 10
```

Comment:
I'd go higher for a default, maybe 32?

Reply:
Hmm, let's split the difference and go for like 20, since that's always getting pretty long for an ident and should give anyone enough space to be unique. We'll see if anyone yells our way about it~